MobileSyncExplorerKotlin/app/AndroidManifest.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.salesforce.samples.mobilesynccompose">
```

Comment:
I think `com.salesforce.androidsmobilesyncexplorerkotlin` would be inline with our template naming scheme.

Reply:
👍 
It will also make my life easier in `template.js` when substituting the user provided package to ours.

Reply:
Yeah MobileSyncCompose was my working title, but I have no problems changing it to what is simplest 👍